File: Unidad_5/C#/Programa7_U5.cs
using System;

class Programa7_U5
{
    static void Main()
    {
        Console.WriteLine("Ingrese el subtotal de la factura:");
        double subtotal = double.Parse(Console.ReadLine());

        double iva = Calcular_IVA(subtotal);
        double total = Calcular_Total(subtotal, iva);

        Console.WriteLine($"Subtotal: ${subtotal}");
        Console.WriteLine($"IVA (16%): ${iva}");
        Console.WriteLine($"Total: ${total}");
    }

    static double Calcular_IVA(double subtotal)
    {
        double porcentaje_IVA = 0.16;
        return subtotal * porcentaje_IVA;
    }

    static double Calcular_Total(double subtotal, double iva)
    {
        return subtotal + iva;
    }
}

File: Unidad_5/Java/Programa7_U5.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa7_u5;

/**
 *
 * @author leebc
 */
import java.util.Scanner;

public class Programa7_U5 {

    public static void main(String[] args) {
   
        Scanner scanner = new Scanner(System.in);

        System.out.println("Ingrese el subtotal de la factura:");
        double subtotal = scanner.nextDouble();

        double iva = calcular_IVA(subtotal);
        double total = calcular_Total(subtotal, iva);

        System.out.println("Subtotal: $" + subtotal);
        System.out.println("IVA (16%): $" + iva);
        System.out.println("Total: $" + total);
    }
    public static double calcular_IVA(double subtotal) {
        double porcentaje_IVA = 0.16;
        return subtotal * porcentaje_IVA;
    }

    public static double calcular_Total(double subtotal, double iva) {
        return subtotal + iva;
    }
}

